Description: update man info
 .
 These updates come from the master branch,
 fc143212ab2659e6d38072ccdd18aec2f24f788f
 made by byang@debian.org on 2018-02-07.
 .
Author: xiao sheng wen <atzlinux@sina.com>

---
Last-Update: 2020-04-23

diff --git a/doc/convert_pgn.6.in b/doc/convert_pgn.6.in
index 1460467..8c3bcc1 100644
--- a/doc/convert_pgn.6.in
+++ b/doc/convert_pgn.6.in
@@ -1,4 +1,4 @@
-.TH convert_pgn 6 "16 july 2009" "" ""
+.TH convert_pgn 6 "16 July 2009" "" ""
 .RE
 .RE
 .PP
@@ -47,7 +47,9 @@ MANUAL by Vern Sun <s5unty@gmail.com>
 .PP
 
 .SH RESOURCES
-Main web site: <http://gmchess.googlecode.com>
+Main website: <https://github.com/lerosua/gmchess>
+.br
+Debian source code repository: https://salsa.debian.org/chinese-team/gmchess
 .RE
 .PP
 
diff --git a/doc/eleeye_engine.6.in b/doc/eleeye_engine.6.in
index d3a1774..9dae375 100644
--- a/doc/eleeye_engine.6.in
+++ b/doc/eleeye_engine.6.in
@@ -1,4 +1,4 @@
-.TH eleeye_engine 6  "16 july 2009" "" ""
+.TH eleeye_engine 6  "16 July 2009" "" ""
 .RS
 .RE
 .RE
diff --git a/doc/gmchess.6.in b/doc/gmchess.6.in
index 52a4108..83291f3 100644
--- a/doc/gmchess.6.in
+++ b/doc/gmchess.6.in
@@ -1,4 +1,4 @@
-.TH GMCHESS 6 "08 july 2009" "gmchess-0.20" 
+.TH GMCHESS 6 "8 February 2018" "gmchess 0.29.6"
 .RS
 .RE
 .RE
@@ -20,9 +20,9 @@
 .fam T
 .fi
 .SH DESCRIPTION
-\fBgmchess\fP is a game of conventional chinese chess. 
-The AI isn't wonderful, but provides a challenging enough game to all but the
-most highly skilled players.
+\fBgmchess\fP is a game of traditional Chinese chess.
+While AI isn't wonderful, it provides a challenging enough game to all but the
+most skilled players.
 .RE
 .PP
 
@@ -32,12 +32,14 @@ most highly skilled players.
 .PP
 
 .SH AUTHOR
-Written by wind <xihels@gmail.com>, lerosua <lerosua@gmail.com>
+Written by wind <xihels@gmail.com> and lerosua <lerosua@gmail.com>.
 .RE
 .PP
 
 .SH RESOURCES
-Main web site: http://gmchess.googlecode.com
+Placeholder GitHub repository: https://github.com/lerosua/gmchess
+.br
+Debian source code repository: https://salsa.debian.org/chinese-team/gmchess
 .RE
 .PP
 
